app/views/admin/categories/edit.html.erb

```diff
@@ -9,7 +9,7 @@
 
       <%= f.input :name, label: t(:new_name, scope: "admin.dashboard.categories"), input_html: { class: "mv" }, required: false %>
 
-      <%= f.submit t(:save, scope: "general"), class:"btn btn--small" %>
+      <%= submit_tag t(:save, scope: "general"), class:"btn btn--small" %>
```

Comment:
Why did you change that? Btw. it would be great to have longer commit messages that explain what you had in mind when doing your changes. So everyone can understand what problem you solved and why you decided to do the way you did. This is not easy and we also try hard and get reminded everytime by our coach ;) But one example is this one:

"add map to header
and add toggle map function
to show and load map only when user is clicking on address

(#5813)"